### PR TITLE
Analytics section and removing texts

### LINKED ISSUE #2376: New style and place for platform statistics
# Goal
* Change the aesthetics of the platform statistics (users, apis etc.) according to the wireframe
* Open issue: How is the color of the area defined? Is it a pre-defined color or based on a branding setting?
* Remove the header "At a glance"
* Also, remove the text sections of the front page entirely.
# Definition of done
* [x] Remove resources related to At a glance
* [x] Remove resources related to Getting started section
* [x] Remove resources related to Reach out to developers
* [x] Remove resources related to Completely open source
* [x] Check that all dependencies about the above resources are removed 
# Wireframe
frontpage1-3

### PR DESCRIPTION
**Pull request by jawidahmadi**
Close #2376

### CODE REVIEW

File: home/client/body/homeBody.less [hunk 1]
```diff
@@ -14,6 +14,7 @@ https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence
   background-size: cover;
   background-position: top center;
   background-color: @brand-primary;
+  margin-bottom: 3.125em;
```
Comment (nnn) [inline home/client/body/homeBody.less]: Two digits after dot are enough.

File: home/client/body/homeBody.html [hunk 2]
```diff
@@ -202,90 +189,6 @@ <h3 class="feature-title">
       </div>
     </div>
  </div>
-
-  <div class="content-section-a">
```
Comment (nnn) [inline home/client/body/homeBody.html]: Remove all related styles for these sections.

File: home/client/body/homeBody.html [hunk 3]
```diff
@@ -24,20 +24,8 @@ <h1 id="site-title" class="text-center">
...
-    <div class="row">
-      <div class="col-lg-12 text-center">
-        <h2 class="feature-section-heading">
-          {{_ "homeBody_feature_section_Statistic" }}
-        </h2>
-        <hr class="primary">
-      </div>
-    </div>
-
+<div class="container" id="features">
```
Comment (nnn) [inline home/client/body/homeBody.html]: This id doesn't fit the content of the container.

File: home/client/body/homeBody.html (hunk 2 above)
Comment (jawidahmadi) [inline home/client/body/homeBody.html]: styles are removed

**Comment by nnn** [comment]
@jawidahmadi check the issue's description, the second part of this task is to create a new style and place for platform statistics based on wireframe. This part is still missing.

**Comment by jawidahmadi** [comment]
@nnn There is a new issue about the second part so, the second part is a separate task.

**Comment by nnn** [comment]
Separated issue is for controlling color of the updated statistics block which should be created in this PR.

File: home/client/body/homeBody.html [hunk 4]
```diff
@@ -311,5 +232,62 @@ <h4 class="text-muted">
...
     </div>
   </div>
+    <!-- Statistics Block Message -->
+    <div class="container" id="statisticswrap">
+    <div class="row">
+     <div class="col-md-4">
+       <div class="panel panel-default">
+         <div class="panel-body">
+           <div class="media">
+             <!-- <div class="media-left media-middle">
```
Comment (brylie) [inline home/client/body/homeBody.html]: Clean up **code** that is commented out.

File: home/client/body/homeBody.html [hunk 5]
```diff
@@ -311,5 +232,62 @@ <h4 class="text-muted">
...
+             </div>
+           </div>
+         </div>
+       </div>
+     </div>
+     <div class="col-md-4">
+       <div class="panel panel-default">
+         <div class="panel-body">
+           <div class="media">
+             <!-- <div class="media-left media-middle">
```
Comment (brylie) [inline home/client/body/homeBody.html]: Clean up **code** that is commented out.

File: home/client/body/homeBody.less [hunk 6]
```diff
@@ -14,6 +14,7 @@ https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence
   background-size: cover;
   background-position: top center;
   background-color: @brand-primary;
+  // margin-bottom: 3.12em;
```
Comment (brylie) [inline home/client/body/homeBody.less]: Clean up **code** that is commented out.

File: home/client/body/homeBody.less [hunk 7]
```diff
@@ -196,10 +145,16 @@ hr.section-heading-spacer {
...
-#contact-us {
-  margin-bottom: 4.37em;
+.media-heading {
+   text-align: center;
+   font-weight: bold;
+}
+
+#statisticswrap {
+  width: 100%;
+  padding-top: 20px;
```
Comment (brylie) [inline home/client/body/homeBody.less]: Use relative values, such as `em`, if possible. Pixel values `px` are very rigid, and might cause problems on various display sizes.

File: home/client/footer/homeFooter.less [hunk 8]
```diff
@@ -11,7 +11,7 @@ footer {
   background-color: #333;
   padding-bottom: 1em;
   padding-top: 2em;
-
+  margin-top: 5px;
```
Comment (brylie) [inline home/client/footer/homeFooter.less]: Use relative values, such as `em`, if possible. Pixel values `px` are very rigid, and might cause problems on various display sizes.

**Comment by nnn** [comment]
@brylie I added some changes to help @jawidahmadi to merge this PR.
If you don't have other suggestions, I will merge.

File: home/client/body/homeBody.html [hunk 9]
```diff
@@ -24,292 +24,70 @@ <h1 id="site-title" class="text-center">
...
-                <h4 class="media-heading">
-                     {{_ "homeBody_featureSection_statistics_apisCount_heading" }}  {{ apisCount }}
-                </h4>
-              </div>
-            </div>
+        <div class="media">
+          <div class="media-body">
+            <h2 class="media-heading">
+              <p> {{ apisCount }} </p>
+              <p> {{_ "homeBody_featureSection_statistics_apisCount_heading" }} </p>
```
Comment (brylie) [inline home/client/body/homeBody.html]: This is a minor issue that can probably be ignored, but why are we putting spaces around the data in the paragraphs?

```html
<p> {{ apisCount }} </p>
```

```html
 <p> {{_ "homeBody_featureSection_statistics_apisCount_heading" }} </p>
```

Also, more often we place child elements on a new line, like so:

```html
<p>
  {{ apisCount }}
</p>
```

```html
<p>
  {{_ "homeBody_featureSection_statistics_apisCount_heading" }}
</p>
```

File: home/client/body/homeBody.html [hunk 10]
```diff
@@ -24,292 +24,70 @@ <h1 id="site-title" class="text-center">
...
-                  <h4 class="media-heading">
-                      {{_ "homeBody_featureSection_statistics_organizationsCount_heading" }}  {{ organizationsCount }}
-                </h4>
-              </div>
-            </div>
+        <div class="media">
+          <div class="media-body" >
+            <h2 class="media-heading">
+              <p> {{ organizationsCount }} </p>
+              <p> {{_ "homeBody_featureSection_statistics_organizationsCount_heading" }} </p>
```
Comment (brylie) [inline home/client/body/homeBody.html]: Same as previous. Why are we putting spaces around the data in the paragraphs?

```html
<p> {{ organizationsCount }} </p>
```

```html
 <p> {{_ "homeBody_featureSection_statistics_organizationsCount_heading" }} </p>
```

Also, more often we place child elements on a new line, like so:

```html
<p>
  {{ organizationsCount }}
</p>
```

```html
<p>
  {{_ "homeBody_featureSection_statistics_organizationsCount_heading" }}
</p>
```

File: home/client/body/homeBody.html [hunk 11]
```diff
@@ -24,292 +24,70 @@ <h1 id="site-title" class="text-center">
...
-              <div class="media-body">
-                   <h4 class="media-heading">
-                       {{_ "homeBody_featureSection_statistics_usersCount_heading" }}   {{ usersCount }}
-                    </h4>
-                </div>
-            </div>
+        <div class="media">
+          <div class="media-body">
+            <h2 class="media-heading">
+              <p> {{ usersCount }} </p>
```
Comment (brylie) [inline home/client/body/homeBody.html]: Again, why the spaces? Perhaps I'm quirky, but these things stand out to me.

**Comment by brylie** [comment]
Really great looking! Lots of code removed :smile: 

Please see my comments about nesting and unnecessary whitespace.

**Pull request merged**